app/src/main/res/layout/dialog_results.xml

```diff
@@ -343,6 +343,7 @@
 
         <include layout="@layout/expanded_results_box"
             android:id="@+id/expandedResultsBox"
+            android:visibility="gone"
```

Comment:
Weren't we using our own font? That should ensure consistent results... and the arrows work correctly for me. I'm guessing they're using some system font that changes across Android versions.